bindings/java/wasmedge-java/src/test/java/org/wasmedge/AstModuleContextTest.java

```diff
@@ -6,6 +6,5 @@ public class AstModuleContextTest extends BaseTest {
 
     @Test
     public void testCreation() {
-        AstModuleContext astContext = new AstModuleContext();
```

Comment:
Why do we still keep an empty test function if this is removed?

Reply:
Removed.